### DISCUSSION TITLE
Didn't receive STX from airdrop

### DISCUSSION THREAD

**Discussion by mattlovelyy** [stacks-network/stacks-core#2958] [Q&A]
Hello I am having a problem. I participated in the STX airdrop on blockchain.com, but no coins were credited to my address. I checked everything, changed wallets, but everywhere I see a zero balance.
My STX address (mainnet): SP2NX3W9E62GM1WZ0Z5AQ9K6FNC2T4V3DJ75KXP4G
(testnet): ST2NX3W9E62GM1WZ0Z5AQ9K6FNC2T4V3DJ4QHZFMX
Help me please
![image](https://user-images.githubusercontent.com/58401338/144069442-d7ce1eae-f393-4e23-b875-8e504c4c7d4b.png)

**Comment by 314159265359879** [comment]
The airdrop is actually carried out by Blockchain.com if you want to be sure about what is going on you should probably contact them. 

From what I gather on the Telegram channel, blockchain.com made an account for you. You can get that seed phrase from your blockchain.com account and then use that to access your funds by using that seed phrase on the Hiro.so Wallet (desktop or wallet for web( extension)).

Some users on telegram found this step by step helpful to get access to the airdropped funds: https://www.youtube.com/watch?v=3s9ukc6HLo8

**Reply by mattlovelyy** [reply]
The point is that I wrote in support of Blockchain.com. There I was advised to contact Stacks, or wait for them to launch this coin on Blockchain.com. It is not clear who is right, but I will write again in support Blockchain. Thanks!

**Comment by mattlovelyy** [comment]
Help me please
